Integrations/integration-ServiceNow.yml

```diff
@@ -0,0 +1,802 @@
+description: IT service management
+configuration:
+- display: ServiceNow URL, in the format https://company.service-now.com/
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Username
+  name: username
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Password
+  name: password
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "True"
+  type: 8
+  required: false
+- display: The query to use when fetching ServiceNow incidents
+  name: sysparm_query
+  defaultvalue: stateNOT IN6,7
+  type: 0
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: How many ServiceNow incidents to fetch each time
+  name: fetch_limit
+  defaultvalue: "10"
+  type: 0
+  required: false
+- display: Trust self-signed certificates
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: ServiceNow API Version (e.g. 'v1')
+  name: api_version
+  defaultvalue: ""
+  type: 0
+  required: false
+script:
+  script: |-
+    var username = params.username;
+    var password = params.password;
+    var serverUrl = params.url;
+
+    if (serverUrl[serverUrl.length - 1] !== '/') {
+        serverUrl += '/';
+    }
+
+    var defaults = {limit:10, fetch_limit:10};
+
+    var Base64 = {
```

Comment:
No need for this... you can just put
Username:
Password: 
Inside the http params (look at algosec or sandblast as ref)